src/components/menu/main-menu.tsx

```diff
@@ -73,7 +73,22 @@ const MainMenu = ({menuItems}: {menuItems: MenuItemType[]}) => {
             )}
           >
             <div className="mb-10 mr-20 text-left">Quick links:</div>
-            <ul className="list-unstyled flex flex-wrap items-center gap-10">
+            <ul className="list-unstyled flex flex-wrap items-center gap-10 @container">
+              <div className="flex flex-wrap gap-10 @sm:gap-5">
```

Comment:
A div can not be inside a ul

Reply:
Doh! I should've know that especially after correcting the People page links 😆  Thanks Mike!